<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1427</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marin Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="file_list_widget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>711</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="image_viewer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_file_name_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>파일명 변경 및 다음 사진으로 이동</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="show_changed_file_name_text_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hello World&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_folder_dir_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더 변경</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ext_combo_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.jpeg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.png</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="composition_combo_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>전신</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>상반신</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>하반신</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디테일</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="detail_combo_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>구도</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>디테일</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>변경된 파일명</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더 내 파일 리스트</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="cloth_type_group_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>의류 종류</string>
    </property>
    <widget class="QRadioButton" name="t_radio_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>상의</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="p_radio_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>겉옷</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="b_radio_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>하의</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="is_model_group_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>모델 유무</string>
    </property>
    <widget class="QCheckBox" name="check_box_no_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>모델 없음</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_m3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>남자 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_f1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>여자 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_f3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>여자 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_f5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>여자 5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_f2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>여자 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_f6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>여자 6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_box_f4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>여자 4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="bg_num_group_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>배경 번호</string>
    </property>
    <widget class="QRadioButton" name="bg_type_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 7</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 8</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bg_type_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>배경 6</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>확장자</string>
    </property>
   </widget>
   <widget class="QListWidget" name="changed_file_list_widget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>711</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>변경 후</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>변경 전</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>색상</string>
    </property>
   </widget>
   <widget class="QComboBox" name="colors_combo_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1427</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>파일</string>
    </property>
   </widget>
   <addaction name="file_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
